src/frontend/Komponenter/Behandling/Brev/BrevUtils.ts

```diff
@@ -105,15 +106,36 @@ export const initFlettefelterMedVerdi = (
         automatiskUtfylt: !!flettefeltStore[flettefeltReferanse.felt],
     }));
 
+function finnDelmal(blokk: BrevmenyBlokk): Delmal | undefined {
+    if (erDelmalBlokk(blokk)) {
+        return blokk.innhold;
+    }
+}
+
+function trekkUtDelmaler(brevStruktur: BrevStruktur) {
```

Comment:
```suggestion
function trekkUtDelmaler(brevStruktur: BrevStruktur) {
```

Gjerne skriv om disse til arrow-functions for å få kodestilen mer lik eksisterende kode :)